.github/workflows/test.yml

```
@@ -66,6 +67,27 @@ jobs:
           dive-image-version: >-
             0.14.0
 
+      - name: 'Params: highest-wasted-bytes, highest-user-wasted-percent, lowest-efficiency'
+        if: always() && steps.pre-setup.outcome == 'success'
+        uses: ./
+        with:
+          image: ${{ env.WASTED_IMAGE }}
+          highest-wasted-bytes: 100MB
+          highest-user-wasted-ratio: 0.5
+          lowest-efficiency: 0.9
```

actionlint (1.7.4)

input "lowest-efficiency" is not defined in action "Dive Action v2" defined at "./". available inputs are "always-comment", "config-file", "dive-image-registry", "dive-image-version", "github-token", "highest-user-wasted-ratio", "highest-wasted-bytes", "image", "lowest-efficiency-ratio"

(action)